haskell/src/Indigo/Contracts/Safelist.hs

```diff
@@ -0,0 +1,73 @@
+-- SPDX-FileCopyrightText: 2020 tqtezos
+-- SPDX-License-Identifier: MIT
```

Comment:
UPDATE: see https://github.com/serokell/tezos-globacap/pull/32

It's fine to use this code here, but the header should be 
```
-- SPDX-FileCopyrightText: 2020 Globacap
-- SPDX-License-Identifier: MPL-2.0
```

Reply:
Changed